src/databricks/labs/ucx/azure/access.py

```diff
@@ -206,9 +206,21 @@ def _create_access_connector_for_storage_account(
             wait_for_provisioning=True,
         )
         self._apply_storage_permission(access_connector.principal_id, role_name, storage_account)
-        return access_connector
 
-    def create_access_connectors_for_storage_accounts(self) -> list[AccessConnector]:
+        container = next(self._azurerm.containers(storage_account.id), None)
```

Comment:
shouldn't we do it for every container instead? you're using only the first container.

Reply:
One container should suffice as we give permissions to the whole storage account.